app/views/layouts/_analytics_header.html.slim

```diff
@@ -0,0 +1,9 @@
+- if track_analytics?
+  /! Google Tag Manager
+  javascript:
```

Comment:
Does this still make a `<script>` tag?

Reply:
Yep, and it doesn't work by just putting in `script`
<img width="536" alt="image" src="https://user-images.githubusercontent.com/13471320/181222692-0640b163-6189-4d40-ae6e-4885c707fd96.png">